lib/config/options/index.ts

```diff
@@ -1248,6 +1248,16 @@ const options: RenovateOptions[] = [
     stage: 'package',
     mergeable: true,
   },
+  {
+    name: 'matchQl',
+    description: 'SQL-like query language for matching',
+    type: 'string',
+    parents: ['packageRules'],
+    stage: 'package',
+    mergeable: true,
+    cli: false,
+    env: false,
```

Comment:
The docs say the feature is _experimental_, but there's no `experimental: true` field in the code.